contracts/0.4.24/Lido.sol

```diff
@@ -51,28 +51,28 @@ contract Lido is ILido, IsContract, Pausable, AragonApp {
     /// @dev default value for maximum number of Ethereum 2.0 validators registered in a single depositBufferedEther call
     uint256 internal constant DEFAULT_MAX_DEPOSITS_PER_CALL = 16;
 
-    bytes32 internal constant FEE_VALUE_POSITION = keccak256("lido.Lido.fee");
-    bytes32 internal constant TREASURY_FEE_VALUE_POSITION = keccak256("lido.Lido.treasuryFee");
-    bytes32 internal constant INSURANCE_FEE_VALUE_POSITION = keccak256("lido.Lido.insuranceFee");
-    bytes32 internal constant NODE_OPERATORS_FEE_VALUE_POSITION = keccak256("lido.Lido.operatorsFee");
+    bytes32 internal constant FEE_VALUE_POSITION = keccak256("fee.lido.lido.eth");
+    bytes32 internal constant TREASURY_FEE_VALUE_POSITION = keccak256("treasuryFee.lido.lido.eth");
+    bytes32 internal constant INSURANCE_FEE_VALUE_POSITION = keccak256("insuranceFee.lido.lido.eth");
+    bytes32 internal constant NODE_OPERATORS_FEE_VALUE_POSITION = keccak256("operatorsFee.lido.lido.eth");
 
-    bytes32 internal constant TOKEN_VALUE_POSITION = keccak256("lido.Lido.token");
-    bytes32 internal constant VALIDATOR_REGISTRATION_VALUE_POSITION = keccak256("lido.Lido.validatorRegistration");
-    bytes32 internal constant ORACLE_VALUE_POSITION = keccak256("lido.Lido.oracle");
-    bytes32 internal constant NODE_OPERATOR_REGISTRY_VALUE_POSITION = keccak256("lido.Lido.nodeOperatorRegistry");
+    bytes32 internal constant TOKEN_VALUE_POSITION = keccak256("token.lido.lido.eth");
+    bytes32 internal constant VALIDATOR_REGISTRATION_VALUE_POSITION = keccak256("validatorRegistration.lido.lido.eth");
```

Comment:
rename `validatorRegistration` to `depositContract`, Everywhere. It's the old deprecated name.

Reply:
Do this with separate commit